src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -57,36 +57,32 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
     setCheckboxElements(checkboxElements.map((cb, i) => (i === index ? changedCheckbox : cb)))
   }
 
+  const styles =
+    layout === 'inline'
+      ? {
+          display: 'inline',
+          marginRight: 10,
+        }
+      : {}
+
   return (
     <>
       <label>
         <Markdown text={label} semanticComponent="noParagraph" />
       </label>
       <Markdown text={description} />
-      {checkboxElements.map((cb: Checkbox, index: number) => {
-        return layout === 'inline' ? (
+      {checkboxElements.map((cb: Checkbox, index: number) => (
+        <Form.Field key={`${index}_${cb.label}`} disabled={!isEditable} style={styles}>
```

Comment:
We can't be using the LESS classes here because it's a plugin, right?

Reply:
We could, but it makes sense to plugin styles contained within their bundle.